01_introduction.html

```diff
@@ -113,5 +113,8 @@
     and <code><a href=http://www.ecma-international.org/ecma-262/5.1/#sec-15.1.2.3>parseFloat</a></code>
     from [[!ECMA-262]] with the exception that non-valid integer
     or float return values are returned as 0 or 0.0.
+
+    <h4>Top Level Browsing Context no longer open</h4>
+    <p>A top level browsing context is <dfn>no longer open</dfn> if it has been <a href="https://html.spec.whatwg.org/#a-browsing-context-is-discarded">discarded</a>
```

Comment:
Can't we just use the definition from HTML, that a browser context has been “discarded”?